src/Breakpoints.js

```diff
@@ -1,4 +1,5 @@
 import debounce from "./debounce";
+import CustomEvent from './CustomEventPolyfill';
```

Comment:
Assuming there's not a babel based way to deal with this, this looks good to me. Just doing a quick Google it looks to me like this specific thing is excluded from babel, so this makes sense. 

Looks like there's a handful of ways to pull this in via NPM as well. If there were one standard, I might say do it that way to reduce code bloat in case _another_ module needs this as well. This one's got a fair number of dependent libraries and lots of downloads, but it doesn't stand out as something everyone's using: https://www.npmjs.com/package/custom-event-polyfill

Reply:
Yeah, agreed - this doesn't seem to fall under the babel or core-js spec so you have to use a polyfill.  I have used that NPM package for the custom event polyfill before, but it seemed like overkill when we don't need IE9 support and the polyfill (from MDN: https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent) is pretty simple and just patches the API which I believe is all that is needed for IE11.